File: math/0171_excel_sheet_column_number/0171_excel_sheet_column_number.py
class Solution(object):
    def titleToNumber(self, s):
        result = 0

        for i in xrange(len(s)):
            result *= 26
            result += ord(s[i]) - ord('A') + 1

        return result

s = "A"
res = Solution().titleToNumber(s)
print(res)

File: math/0171_excel_sheet_column_number/0171_excel_sheet_column_number.go
package main 

import "fmt"

func titleToNumber(s string) int {
	res := 0

	for i:=0; i< len(s); i ++ {
		temp := int(s[i] - 'A' + 1)
		res = res*26 + temp
	}

	return res
}

func main() {
	s := "A"
	res := titleToNumber(s)
	fmt.Println(res)
}